mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -18,14 +18,16 @@
     <!-- No Open Tabs Message Description -->
     <string name="no_open_tabs_description">Your open tabs will be shown here.</string>
 
+    <!-- About content. The first parameter is the name of the application. (For example: Fenix) -->
+    <string name="about_content">%1$s is produced by Mozilla.</string>
+
     <!-- Private Browsing -->
     <!-- Title for private session option -->
     <string name="private_browsing_title">You’re in a private session</string>
     <!-- Explanation for private browsing displayed to users on home view when they first enable private mode
         The first parameter is the name of the app defined in app_name (for example: Fenix) -->
-    <string name="private_browsing_placeholder_description">
-         %1$s clears your search and browsing history when you quit the app or close all private tabs. While this doesn’t make you anonymous to websites or your internet service provider, it makes it
-        easier to keep what you do online private from anyone else who uses this device.
+    <string name="private_browsing_placeholder_description_2">
+         %1$s clears your search and browsing history from private tabs when you close them or quit the app. While this doesn\'t make you anonymous to websites or your internet service provider, it makes it easier to keep what you do online from anyone else who uses this device.
```

Comment:
@mcarare 
Please use an apostrophe `‘` instead of a straight single quote (which gets escaped). That's also inconsistent with the rest of the product (and other products).

@betsymi 
Betsy, I see you proposed the new string in the original PR. Is the change from "keep X private from anyone" to "keep X from anyone" wanted? The latter sounds a bit strange to me.

Reply:
I will confirm with @betsymi tomorrow and update the apostrophe!

Reply:
We make keep the private if that helps with clarity. 

> Firefox Nightly clears your search and browsing history from private tabs when you close them or quit the app. While this doesn't make you anonymous to websites or your internet service provider, it makes it easier to keep what you do online private from anyone else who uses this device.

Reply:
I'm fine with both, mostly wanted to check it was wanted.

Reply:
It has been updated!